### ISSUE TITLE
Add `get_area_from_y_axis()` function to `CoordinateSystem`

### ISSUE THREAD

**Issue by JeroSQ** [ManimCommunity/manim#2583]
## Description of proposed feature
A function that allows you to get the area of a function bounded by the y axis. An integral from y0 to y1 (I don't know if there's a proper name for this). Like such:
![example](https://user-images.githubusercontent.com/68753500/155862707-2eef4756-e9c6-4b79-80eb-f38525b05d38.gif)

## How can the new feature be used?
Would improve the graphing tool and add functionality.

## Additional comments
This may be regarded as too specific. Let me know what you think. Also the function name doesn't have to be the one proposed in the title of the issue.

**Comment by nikolim** [comment]
Hi @JeroSQ
This sounds like a great feature and I would like to work on it with colleagues. 
Could you assign the issue to me?

**Comment by JeroSQ** [comment]
I can't assign issues but feel free to work on it. Devs need to decide if this feature can be implemented into the library or into manim-contrib

**Comment by Darylgolden** [comment]
I don't think this is too specific to be implemented, but it should be implemented in by rewriting `get_area` such that it can be used on any axis, rather than implementing a different logic for the x and y axes.
